package.json

```diff
@@ -65,21 +67,24 @@
     "@types/pluralize": "0.0.29",
     "@types/sinon": "10.0.4",
     "@types/vscode": "1.62.0",
+    "@typescript-eslint/eslint-plugin": "^5.7.0",
+    "@typescript-eslint/parser": "^5.7.0",
     "@vscode/test-electron": "1.6.2",
     "all-contributors-cli": "6.20.0",
     "chai": "4.3.4",
+    "eslint": "^8.5.0",
+    "eslint-config-prettier": "^8.3.0",
     "glob": "7.2.0",
     "husky": "7.0.4",
     "hygen": "6.1.0",
     "jest": "27.4.2",
+    "lint-staged": "^12.1.3",
```

Comment:
🍰 I prefer to omit the caret and install exact versions of packages. That leaves no room for surprises. The JS world isn't very SemVer friendly, so I had surprises in the past with flexible versions being installed.

We have good tests and dependabot regularly makes sure we keep up-to-date, so that's not an issue.